pandas/tests/reshape/test_concat.py

```diff
@@ -765,13 +765,15 @@ def test_concat_join_axes_deprecated(self, axis):
         )
 
         expected = pd.concat([one, two], axis=1, sort=False).reindex(index=two.index)
-        result = pd.concat([one, two], axis=1, sort=False, join_axes=[two.index])
+        with tm.assert_produces_warning(FutureWarning):
```

Comment:
Where are these warnings coming from?

Reply:
join_axes is deprecated

Reply:
And this failure wasn't caught earlier because the numpydev build is the only one that fails on warnings, and that was offline for a bit? If so, feel free to merge.

Reply:
That's my understanding, yes

Reply:
`join_axes` is already deprecated for a few months, so there's probably another reason ..